src/Extensions/CodeFormatter.hpp

```diff
@@ -91,11 +91,13 @@ class CodeFormatter : public QObject
     QVariant getSetting(const QString &key) const;
 
   protected:
-    QCodeEditor *editor;
-    QString lang;
-    bool selectionOnly;
-    bool logOnNoChange;
-    int cursorPos, cursorLine, cursorCol, anchorPos, anchorLine, anchorCol;
+    QCodeEditor *editor;                             // NOLINT: cppguidelines do not support use of protected members
```

Comment:
Protected data members is bad. See [this](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-protected)